src/react-components/home/HomePage.scss

```diff
@@ -33,6 +34,17 @@
   }
 }
 
+:local(.hamburger) {
+  pointer-events: auto;
+  cursor: pointer;
```

Comment:
<IconButton /> already has this in the styling.  Any reason why you're using it here?

Reply:
That's correct, the style is already applied on mobile-nav-wrapper. Removing.